bpf/bpf_lxc.c

```diff
@@ -236,6 +240,8 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 	case CT_REPLY:
 		policy_mark_skip(skb);
 
+		if (ct_state.proxy_port)
+			verdict = ct_state.proxy_port;
```

Comment:
This part should be unnecessary. On egress reply, if the connection is already going through the proxy then the connection will be between the host and the endpoint, so we don't need to do special handling to redirect to proxy.